packages/expo-modules-core/ios/Core/Convertibles/Convertibles+Color.swift

```diff
@@ -1,6 +1,10 @@
 // Copyright 2022-present 650 Industries. All rights reserved.
 
 extension UIColor: Convertible {
+  private static func resolveNamedColor(name: String) -> UIColor? {
```

Comment:
Similarly, it would be great to move this function out of the extension. This extension is to provide conformance to `Convertible` protocol, so anything else is not necessary here.